src/ontology/config/omim_exclusions.tsv

```diff
@@ -210,6 +210,243 @@ OMIM:618079	low density lipoprotein cholesterol level quantitative trait locus 8
+OMIM:157960	moloney leukemia virus integration site 2, mouse, homolog of	MONDO:excludeGene	FALSE
+OMIM:158050	monkey red blood cell receptor	MONDO:excludeGene	FALSE
+OMIM:165210	oncogene bmyc	MONDO:excludeGene	FALSE
+OMIM:165290	oncogene rmyc	MONDO:excludeGene	FALSE
+OMIM:168710	parotid proline-rich salivary protein pc	MONDO:excludeGene	FALSE
+OMIM:182269	small proline-rich protein 2c, pseudogene	MONDO:excludeGene	FALSE
+OMIM:190445	triiodothyronine receptor auxiliary protein	MONDO:excludeGene	FALSE
+OMIM:194490	zinc finger protein 1	MONDO:excludeGene	FALSE
+OMIM:194630	zinc finger protein 123, pseudogene	MONDO:excludeGene	FALSE
+OMIM:300113	X-linked B cell surface antigen, mouse, homolog-like 1	MONDO:excludeGene	FALSE
+OMIM:301063	XAGE5	MONDO:excludeGene	FALSE
+OMIM:301084	TCEANC	MONDO:excludeGene	FALSE
+OMIM:301085	ADGRG4	MONDO:excludeGene	FALSE
+OMIM:301086	MOSPD2	MONDO:excludeGene	FALSE
+OMIM:301087	FGF13AS1	MONDO:excludeGene	FALSE
+OMIM:301088	CMC4	MONDO:excludeGene	FALSE
+OMIM:309050	lutheran suppressor, X-linked	MONDO:excludeGene	FALSE
+OMIM:600149	glycerol kinase 3 pseudogene	MONDO:excludeGene	FALSE
+OMIM:600703	ribosomal protein s1-like	MONDO:excludeGene	FALSE
+OMIM:600720	nitric oxide synthase 2 pseudogene 1	MONDO:excludeGene	FALSE
+OMIM:600762	human papillomavirus e5 central sequence-like 1	MONDO:excludeGene	FALSE
+OMIM:600841	eukaryotic translation elongation factor 1 alpha-1-like 14	MONDO:excludeGene	FALSE
+OMIM:601079	zrsr2 pseudogene 1	MONDO:excludeGene	FALSE
+OMIM:601151	dead/h-box helicase 12, pseudogene	MONDO:excludeGene	FALSE
+OMIM:601221	tissue-specific extinguisher 3	MONDO:excludeGene	FALSE
+OMIM:602011	suppressor of tumorigenicity 11	MONDO:excludeGene	FALSE
+OMIM:603209	cytidine monophospho-n-acetylneuraminic acid hydroxylase, pseudogene	MONDO:excludeGene	FALSE
+OMIM:603282	zinc finger protein 204	MONDO:excludeGene	FALSE
+OMIM:603738	basic transcription factor 3 pseudogene 12	MONDO:excludeGene	FALSE
+OMIM:603739	basic transcription factor 3 pseudogene 13	MONDO:excludeGene	FALSE
+OMIM:603976	zinc finger protein 94	MONDO:excludeGene	FALSE
+OMIM:603979	zinc finger protein 97	MONDO:excludeGene	FALSE
+OMIM:603985	zinc finger protein 103	MONDO:excludeGene	FALSE
+OMIM:603987	zinc finger protein 105	MONDO:excludeGene	FALSE
+OMIM:603988	zinc finger protein 106	MONDO:excludeGene	FALSE
+OMIM:603991	zinc finger protein 109	MONDO:excludeGene	FALSE
+OMIM:603992	zinc finger protein 110	MONDO:excludeGene	FALSE
+OMIM:603993	zinc finger protein 111	MONDO:excludeGene	FALSE
+OMIM:603995	zinc finger protein 113	MONDO:excludeGene	FALSE
+OMIM:603997	zinc finger protein 118	MONDO:excludeGene	FALSE
+OMIM:603998	zinc finger protein 119	MONDO:excludeGene	FALSE
+OMIM:604000	zinc finger protein 122	MONDO:excludeGene	FALSE
+OMIM:604301	oculomedin	MONDO:excludeGene	FALSE
+OMIM:604754	zinc finger protein 271, pseudogene	MONDO:excludeGene	FALSE
+OMIM:604966	protocadherin-alpha gene cluster	MONDO:excludeGene	FALSE

```

Comment:
Remember that you do not _have_ to exclude these 1 by 1. @joeflack4 can you give @sabrinatoro and example on how to exclude all terms that end with `gene cluster`, `protein`, `pseudogene` and start with `blood group`?

Reply:
@matentzn I know... :-) but since I was going through the list, it was easy to just add it "in bulk". 
I will work on "exclusion rules" in the upcoming few weeks (hopefully before the end of the year)

Reply:
Sure. @sabrinatoro I just realized I need to add this to the list of documentation I need to do.

The exclusion tables support regular expressions. I often like to use sites like https://www.regexpal.com/ to check my syntax. But in this case it's a simple regex, e.g.: `.*gene cluster` should match any line of text that ends with 'gene cluster'.

In order to use a regex with the exclusion tables, it needs `REGEX:` before the regex. So in the `term_label` column, you will want it to say `REGEX:.*gene cluster`, and then it will find all terms with a label that matches that regex.